modules/apps/calendar/calendar-web-test/src/testFunctional/tests/ManageCalendar.testcase

```diff
@@ -183,39 +180,37 @@ definition {
 			calendarType = "Current Site Calendars",
 			myCalendarColor = "668CD9");
 
-		CalendarScheduler.viewEventColor(
-			eventBackgroundColorRgb = "(208, 219, 240)",
-			eventFontColorRgb = "(104, 142, 217)",
-			eventTitle = "Calendar Event Regular Title");
+		AssertCssValue(
+			key_eventTitle = "Calendar Event Regular Title",
+			locator1 = "Calendar#SCHEDULER_EVENT",
+			locator2 = "background-color",
+			value1 = "rgba(102, 140, 217, 0.7)");
+
+		AssertCssValue(
+			key_eventTitle = "Calendar Event Regular Title",
+			locator1 = "Calendar#SCHEDULER_EVENT_CONTENT",
+			locator2 = "color",
+			value1 = "rgba(39, 40, 51, 1)");
 
 		ApplicationsMenu.gotoPortlet(
 			category = "Users",
 			panel = "Control Panel",
 			portlet = "Users and Organizations");
 
-		User.addCP(
+		JSONUser.addUser(
 			userEmailAddress = "userea@liferay.com",
 			userFirstName = "userfn",
 			userLastName = "userln",
 			userScreenName = "usersn");
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Users",
-			panel = "Control Panel",
-			portlet = "Users and Organizations");
-
-		User.editPasswordCP(
-			userEmailAddress = "userea@liferay.com",
-			userScreenName = "usersn");
+		JSONUser.setFirstPassword(
+			agreeToTermsAndAnswerReminderQuery = "true",
+			requireReset = "false",
+			userEmailAddress = "userea@liferay.com");
```

Comment:
@Luizcarlosqueiroz don't forget to add the following property when using this macro: `property custom.properties = "jsonws.web.service.paths.excludes=";`